bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -152,7 +154,12 @@ class Base extends Component {
       meetingExist,
     } = this.props;
 
-    if (meetingExisted && !meetingExist) {
+    if (ejected) {
+      const { ejectedReason } = ejected;
+      return (<MeetingEnded code={ejectedReason} />);
```

Comment:
Looking at the code it seems like here we should Session.set(codeError, ejectedReason) and `return` in the next block where we check for meetingExisted and leave audio. Otherwise we will not get to that other block. Comments are welcome